.github/ISSUE_TEMPLATE/user-support.yml

```diff
@@ -0,0 +1,74 @@
+name: User Support
+description: Get support for an issue on Arjuna
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        If you notice an issue on Arjuna, please *first search the existing issues* and ensure that it has not been reported. If you notice a similar example, please *comment on that issue*.
+
+        Tips for getting help
+        - Quote your shell/log files with ``` `backticks` ```. It makes reading them **much** easier
+        - Include **both** the output of a command **and** the command itself
+        - A minimal working example is a huge plus, we do not debug user code
+        - Unfamiliar with Markdown? See Github's [Markdown Guide](https://guides.github.com/features/mastering-markdown/)
+
+        Requests for support with insufficient information will be tagged `incomplete` and closed. If this happens to you, please add additional information and re-open the issue
+
+  - type: input
+    attributes:
+      label: Your Name
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Andrew ID
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Where it Happened
+      description: Provide the JobIds and Nodes where the problem occurred.
+      placeholder: |
+        JobID: 129999
+        Nodes: c001
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: What Happened?
+      description: What behavior was observed and what did you expect?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: What specific commands are needed to reproduce the issue?
+      render: Shell
+      placeholder: |
+        conda activate ...
+        sbatch ...
+  - type: textarea
+    attributes:
+      label: Job Submission Script
+      description: Provide the job submission script (If any)
```

Comment:
do we want location of or provide the script? don't care too much either way, up to you, just want to make sure we're being precise with language. If you meant actually provide the script, then try to be a little more explicit.